java-checks/src/main/java/org/sonar/java/checks/AnonymousClassShouldBeLambdaCheck.java

```diff
@@ -20,12 +20,18 @@
 package org.sonar.java.checks;
 
 import com.google.common.collect.Lists;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
 import org.sonar.check.Rule;
 import org.sonar.java.JavaVersionAwareVisitor;
+import org.sonar.java.resolve.ClassJavaType;
 import org.sonar.java.resolve.JavaSymbol;
+import org.sonar.java.resolve.JavaType;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'org.sonar.java.resolve.JavaType'. ([squid:UselessImportCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.java%3Ajava&pullRequest=1946&issues=AWJOVuCC2-ApEdl5p-7K&open=AWJOVuCC2-ApEdl5p-7K)

[ISSUE_KEY]: AWJOVuCC2-ApEdl5p-7K